src/common/data-access/EntityEditabilityResolver.ts

```diff
@@ -0,0 +1,5 @@
+export default interface EntityEditabilityResolver {
+
+	isEditable(): Promise<boolean>;
```

Comment:
Granted it would not be of much use ATM but not passing an entity (identifier) when figuring out its editability is... interesting.

Reply:
added!